semana5/projeto-semana5/projetowhats/src/components/MainBox/MainBox.js

```diff
@@ -0,0 +1,74 @@
+import React from 'react'
+import './MainBox.css'
+
+class MainBox extends React.Component {
+    constructor(props){
+        super(props);
+        this.state={
+            msgEnviada: false,
+            usuario: "",
+            msg: ""
+        }
+        
+
+    }
+
+    aoClicar = () =>{
+        this.setState({
+            msgEnviada: true,
+            
+        })
+    }
+
+    mostraMsg = () => {
+        
+        if (this.state.msgEnviada ==  true){
+            return(
+                <div className="box-msg">
+                    <p><strong>{this.state.usuario}</strong>:</p>
+                    <p>{this.state.msg}</p>
+                </div>
+
+                
+            )
+
+        }
+       
+    }
```

Comment:
Entendo o que você quis fazer aqui, mas dessa forma o app sempre vai mostrar uma só mensagem, com aquilo que o usuário estiver digitando no momento. Uma vez que o usuário aperta enviar, o this.state.msgEnviada é sempre true.